dev/build/windows/makecoq_mingw.sh

```diff
@@ -1421,12 +1421,12 @@ function make_addon_ltac2 {
 # A function definition plugin
 
 function make_addon_equations {
-  Equations_SHA=$(git ls-remote "$Equations_CI_GITURL" "refs/heads/$Equations_CI_BRANCH" | cut -f 1)
+  Equations_SHA=$(git ls-remote "$Equations_CI_GITURL" "refs/heads/$Equations_CI_REF" | cut -f 1)
   if [[ "$Equations_SHA" == "" ]]; then
-      # $Equations_CI_BRANCH must have been a tag and not a branch
-      Equations_SHA="$Equations_CI_BRANCH"
+      # $Equations_CI_REF must have been a tag / commit and not a branch
+      Equations_SHA="$Equations_CI_REF"
   fi
-  if build_prep "${Equations_CI_GITURL}/archive" "$Equations_SHA" zip 1 "Equations-$Equations_SHA"; then
+  if build_prep "$Equations_CI_ARCHIVEURL" "$Equations_SHA" zip 1 "Equations-$Equations_SHA"; then
     # Note: PATH is autmatically saved/restored by build_prep / build_post
     PATH=$COQBIN:$PATH
     logn coq_makefile ${COQBIN}coq_makefile -f _CoqProject -o Makefile
```

Comment:
The amount of boilerplate in these windows scripts is a bit suboptimal.

Reply:
This boilerplate is going away in @MSoegtropIMC's PR.